assets/components/SelectInput.jsx

```diff
@@ -29,7 +29,7 @@ SelectInput.propTypes = {
     name: PropTypes.string.isRequired,
     label: PropTypes.string.isRequired,
     onChange: PropTypes.func.isRequired,
-    defaultOption: PropTypes.string,
+    defaultOption: PropTypes.bool,
```

Comment:
this won't allow you to set default value, maybe the condition should be like `{defaultOption != null && ...}`